app/views/events/_new.html.erb

```diff
@@ -7,25 +7,24 @@
         </button>
         <h4 class="modal-title">Event Details</h4>
         <%= form_for(@modal_event) do |f| %>
-            <div class="form-group">
-              <%= f.label :name, "Event Name", class: "control-label" %>
-              <%= f.text_field :name, class: "form-control" %>
-            </div>
+          <div class="form-group">
+            <%= f.label :name, "Event Name", class: "control-label" %>
+            <%= f.text_field :name, class: "form-control" %>
+          </div>
 
-            <div class="form-group">
-              <%= f.label :start_on, "Start date", class: "control-label" %>
-              <%= f.text_field :start_on, class: "form-control datepicker" %>
-            </div>
+          <div class="form-group">
+            <%= f.label :start_on, "Start date", class: "control-label" %>
```

Comment:
There is a bit of inconsistency in capitalization at the moment, where "Event Name" has title capitalization, but "Start date" and "End date" do not.